bin/runner.js

```diff
@@ -232,7 +232,7 @@ function elmPathToModule(elmRoot, sourceDirs, testsPath) {
   };
 }
 
-function findModule(elmRoot, sourceDirs, pathName) {
+function findModule(elmRoot, sourceDirs = ["./src"], pathName) {
```

Comment:
`findModule()` is called in a manner of `findModule(root, model["source-directories"], pathName)`, which leads to nullish referencing inside the function in case of package Elm project.
I've chosen the path of adding fallback `sourceDirs` for minimal impact/low friction in mind.